addons/website/static/src/xml/website.res_config_settings.xml

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<templates id="template" xml:space="preserve">
-<t t-name="website.res_config_settings.cookies_modal_main">
-    <main role="alert">
-        <p>
-            <b>Cookie bars may significantly impair the experience</b> of
-            your visitors. We recommend you avoid them unless you have
-            verified with a legal advisor that you absolutely need cookie
-            consent in your country.
-        </p>
-        <p>
-            For session cookies, authentification and analytics,
-            <b>you do not need to ask for the consent</b> (see e.g. Opinion
-            04/2012 on Cookie Consent Exemption by the EU Art.29 WP).
-        </p>
-    </main>
-</t>
-</templates>
```

Comment:
Should this file first be moved in the dedicated move commit to preserve some history ? :+1: